### PR TITLE
Update item.py

### PR DESCRIPTION
**Pull request by Ryne777**
Double properties in last comment

### CODE REVIEW

**Comment by Ryne777** [comment]
Fix comments in item schema

**Comment by adamrt** [comment]
@Ryne777 Looks like a typo in the fix as well.

File: {{cookiecutter.project_slug}}/backend/app/app/schemas/item.py [hunk 1]
```diff
@@ -34,6 +34,6 @@ class Item(ItemInDBBase):
     pass
 
 
-# Properties properties stored in DB
+# AdAdditional properties stored in DB
```
Comment (StephenBrown2) [inline {{cookiecutter.project_slug}}/backend/app/app/schemas/item.py:37]:
```suggestion
# Additional properties stored in DB
```

**Comment by tiangolo** [comment]
Thanks! This was handled in another PR, so I'll now pass on this one, but thanks for the interest! :coffee:

**Pull request closed without merging**